src/test/java/seedu/address/logic/commands/ViewRentalCommandTest.java

```diff
@@ -82,8 +80,9 @@ public void execute_validIndexFilteredList_success() {
         expectedModel.setLastViewedClient(clientToViewRental);
         expectedModel.updateVisibleRentalInformationList(rentalInformationOfClient);
 
+        // TODO: Zach to take a look. Its working now, simply by commenting out line 85.
```

Comment:
Thanks for leaving this `TODO` comment as we discussed! I'll take a look at it after this PR is merged.